source/src/python/PyRosetta/README.md

```diff
@@ -1,87 +1,30 @@
+
 PyRosetta 4
 ===========
 
-Building with external Rosetta libraries
-----------------------------------------
-
-PyRosetta may be linked against externally compiled Rosetta shared libraries.
-This may be used to suppport "extras" builds or debug builds of the shared
-libraries with release-mode bindings. To support external links, configure &
-build via cmake and clang, then specify `--external-link` to `build.py`.
-
-For example:
-````
-export CC=`which clang`
-export CXX=`which clang++`
-
-cd `git rev-parse --show-toplevel`
-
-pushd source/cmake
-./make_project.py all
-
-pushd build_debug
-cmake -G ninja && ninja
-
-popd
-popd
-
-pushd source/src/python/PyRosetta
-python build.py --external-link debug
-````
-
-Building PyRosetta under Anaconda python
-----------------------------------------
-If you use Anaconda python (https://www.continuum.io) and would like to use
-it to build PyRosetta, you will need to make the build system aware of the
-location of the appropriate C headers and library. These values are specified
-using the `python-include-dir` and `python-lib` command line flags when calling
-`build.py`. For example, if Anaconda 3 is installed in the default location,
-the command to run `build.py` would be:
+Building from source
```

Comment:
Is it worth a note about the licensing situation (e.g. a commercial license for Rosetta doesn't come with a commercial license for PyRosetta?)

Reply:
i guess stating this one more time will hurt, - i will add our standard declaimer here.